server/api/slack.js

```diff
@@ -83,7 +83,7 @@ const oauthAccess = code => {
     client_secret: process.env.SLACK_CLIENT_SECRET
   };
 
-  return axios.get("https://slack.com/api/oauth.access", {
+  return axios.get("https://slack.com/api/oauth.v2.access", {
```

Comment:
Do we need to touch any of these calls?

https://github.com/datadotworld/slack-app/blob/c45330f69adfa19de5098049e183e18316fbf1fb/server/routes/__test__/auth.test.js#L31 

is that a request to our app's oauth endpoint?

Reply:
Yeah, it's a call to our own oauth endpoint, so we don't need to change it.